packages/auth/src/ResetPassword/ResetPasswordForm.tsx

```diff
@@ -25,15 +25,16 @@ export interface ResetPasswordFormProps {
 export const ResetPasswordForm: FC<ResetPasswordFormProps> = (props) => {
   const { renderer, userId, token } = props;
   const { t } = useT();
-  const { loading, error, resetPassword } = useAuth(({ forgotPasswordState }) => forgotPasswordState);
+  const { loading, error, resetPassword, passwordConfig } = useAuth(({ forgotPasswordState }) => forgotPasswordState);
 
   if (renderer) {
     return renderer(omitProps(props, ['renderer']));
   }
+
   return (
     <Formik
       validationSchema={validateSchema({
-        password: validatePassword(t),
+        password: validatePasswordUsingOWASP(passwordConfig),
```

Comment:
What if the password config is null?

Reply:
it will use the default owasp configuration